### PR TITLE
Ensime section update

### PR DESCRIPTION
**Pull request by rajeshpg**
update ensime section with latest version of ensime-sbt and instructions to create .ensime file

### CODE REVIEW

File: documentation/manual/gettingStarted/IDE.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ Follow the installation instructions at <https://github.com/ensime/ensime-emacs>
 Edit your project/plugins.sbt file, and add the following line (you should first check <https://github.com/ensime/ensime-sbt> for the latest version of the plugin):
 
 ```scala
-addSbtPlugin("org.ensime" % "ensime-sbt" % "0.2.3")
+addSbtPlugin("org.ensime" % "ensime-sbt" % "2.0.1")
```
Comment (PromanSEW) [inline documentation/manual/gettingStarted/IDE.md]:
```scala
addSbtPlugin("org.ensime" % "sbt-ensime" % "2.0.1")
```
http://ensime.org/build_tools/sbt/#install

Comment (rajeshpg) [reply documentation/manual/gettingStarted/IDE.md]: @PromanSEW thanks for pointing this out.
I have updated the doc.

File: documentation/manual/gettingStarted/IDE.md [hunk 2]
```diff
@@ -217,5 +192,5 @@ Check out the ENSIME README at <https://github.com/ensime/ensime-emacs>. If you
 1. Eclipse Scala IDE: <http://scala-ide.org/>
 2. NetBeans Scala Plugin: <https://github.com/dcaoyuan/nbscala>
 3. IntelliJ IDEA Scala Plugin: <https://blog.jetbrains.com/scala/>
-4. ENSIME - Scala IDE Mode for Emacs: <https://github.com/aemoncannon/ensime>
+4. ENSIME - Scala IDE Mode for Emacs: <https://github.com/ensime/ensime-emacs>
 (see below for ENSIME/Play instructions)
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/IDE.md]: "see below" what? This is the end of file.

Comment (marcospereira) [reply documentation/manual/gettingStarted/IDE.md]: I understand this was already here, but I think you can just remove this line since it is confusing.

Comment (rajeshpg) [reply documentation/manual/gettingStarted/IDE.md]: i will remove the line and commit.

**Comment by marcospereira** [comment]
Backport to 2.6.x: 5dc8ef7625d00c606df5788e131764c29f1dcd23

**Comment by marcospereira** [comment]
Thank you very much, @rajeshpg!

**Pull request merged**